dali/core/byte_io_test.cc

```diff
@@ -59,6 +60,28 @@ TEST(byte_io, read_value_float) {
   EXPECT_EQ(1.0f, ReadValueLE<float>(data_le));
 }
 
+TEST(byte_io, read_value_unsigned_int_input_stream) {
+  const uint8_t data[] = {0x04, 0xd2, 0x1e, 0x61};  // dec 7777 = hex 0x1E61
```

Comment:
```suggestion
  const uint8_t data_be[] = {0x04, 0xd2, 0x1e, 0x61};  // dec 7777 = hex 0x1E61
```
x86, ARM (and most computers, in fact) natively work with LE, so marking data as "LE" and not marking "BE" as such is misleading.

Reply:
Done